src/pygsdata/coordinates.py

```diff
@@ -19,7 +19,7 @@ def moon_azel(times: apt.Time, obs_location: apc.EarthLocation) -> np.ndarray:
 
 def sun_azel(times: apt.Time, obs_location: apc.EarthLocation) -> np.ndarray:
     """Get local coordinates of the Sun using Astropy."""
-    sun = apc.get_moon(times).transform_to(apc.AltAz(location=obs_location))
+    sun = apc.get_sun(times).transform_to(apc.AltAz(location=obs_location))
```

Comment:
This looks good.

Reply:
But I think it might be useful to update both the functions to get_body('moon') and get_body('sun')

Reply:
I agree -- I'll do that now.